### PR TITLE
[#157245744] Add endpoints for PagoPA proxy

### PR DESCRIPTION
**Pull request by lussoluca**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Remember to fix the PR title by removing WIP wording when ready
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#157245744](https://www.pivotaltracker.com/story/show/157245744): Esporre gli endpoint di pagoPA proxy all'app

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: api_proxy.yaml [hunk 1]
```diff
@@ -242,27 +242,114 @@ paths:
...
+          examples:
+            application/json:
+              importoSingoloVersamento: 200,
+              codiceContestoPagamento: "ABC123"
+        "400":
+          description: Invalid input or PagoPA response
+        "500":
+          description: PagoPA services are not available or request is rejected
+  "/payment-activations":
+    x-swagger-drouter-controller: PagoPAProxyController
```
Comment (cloudify) [inline api_proxy.yaml]: typo? `drouter`

File: api_proxy.yaml [hunk 2]
```diff
@@ -242,27 +242,114 @@ paths:
           description: There was an error in retrieving the user session data.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/payment-requests/{rptId}":
```
Comment (cloudify) [inline api_proxy.yaml]: too bad there's no way to "include" another openapi spec

File: api_proxy.yaml [hunk 3]
```diff
@@ -271,18 +358,34 @@ definitions:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.40.0/api/definitions.yaml#/OrganizationName"
   PaginationResponse:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.40.0/api/definitions.yaml#/PaginationResponse"
+  PaymentActivationsGetResponse:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/italia-pagopa-proxy/skeleton_with_soap_server/api_pagopa.yaml#/definitions/PaymentActivationsGetResponse"
```
Comment (cloudify) [inline api_proxy.yaml]: we should wait to merge this PR until https://github.com/teamdigitale/italia-pagopa-proxy/pull/46 gets merged

File: api_proxy.yaml (hunk 1 above)
Comment (lussoluca) [reply api_proxy.yaml]: good catch! Is a copy&paste from https://github.com/teamdigitale/italia-pagopa-proxy/blob/skeleton_with_soap_server/api_pagopa.yaml#L32, I'll comment the relative PR

File: api_proxy.yaml [hunk 4]
```diff
@@ -242,17 +242,104 @@ paths:
...
+          description: Invalid input
+        "404":
+          description: Activation status not found
+        "500":
+          description: Unavailable service
 definitions:
   BlockedInboxOrChannels:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.47.1/api/definitions.yaml#/BlockedInboxOrChannels"
+  CodiceContestoPagamento:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/italia-pagopa-proxy/integration_with_typed_api/api_pagopa.yaml#/definitions/CodiceContestoPagamento"
```
Comment (cloudify) [inline api_proxy.yaml]: we have merged the specs to `master` we need to change the reference

**Pull request merged**